lib/gc/Transforms/IterativeTilingAndFusion.cpp

```diff
@@ -647,11 +648,18 @@ static LogicalResult isTiledOpInLoop(Operation *targetOp) {
 
 using OpTileSizeMap = std::unordered_map<std::string, SmallVector<int64_t>>;
 
+struct defaultTileConfig {
+  // OpTy-to-TileSize mapping
+  OpTileSizeMap tsMap;
+  // ND-tile size
+  int ndTile;
```

Comment:
```suggestion
  int ndTileSize;
```
Does negative tile size make any sense?

Reply:
Thanks for point out! `unsigned` is enough here.